patches/third_party-blink-renderer-core-core_idl_files.gni.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/core/core_idl_files.gni b/third_party/blink/renderer/core/core_idl_files.gni
+index cbefd9c9bda5dc56ec967616a8182529066c8d2e..af3f7521bffb76498621092e3757da9a65e4070b 100644
+--- a/third_party/blink/renderer/core/core_idl_files.gni
++++ b/third_party/blink/renderer/core/core_idl_files.gni
+@@ -193,6 +193,7 @@ core_idl_files =
+                     "frame/history.idl",
+                     "frame/intervention_report_body.idl",
+                     "frame/location.idl",
++                    "../../../../brave/third_party/blink/renderer/core/frame/navigator_brave.idl",
```

Comment:
might be possible to include another GNI that has all the brave specific ones (which for now, would just be `navigator_brave.idl`)

Reply:
we shouldn't add things like this because it's not extensible. You should use `core_idl_files += brave_core_idl_files` where `brave_core_idl_files` is defined in `//brave/third_party/blink/renderer/core/cord_idl_files.gni` and imported in `//brave/build/config/brave_build.gni`

Reply:
but per above I think this needs to go in `module_idl_files.gni`